src/layer/dataset.ts

```diff
@@ -209,6 +209,20 @@ export const DATASET_AWS_DEM: Dataset = {
   shWmsEvalsource: 'DEM',
   shProcessingApiDatasourceAbbreviation: 'DEM',
   datasetParametersType: null,
+  shServiceHostname: 'https://services.sentinel-hub.com/',
+  searchIndexUrl: null,
+  findDatesUTCUrl: null,
+  orbitTimeMinutes: null,
+  minDate: null,
+  maxDate: null,
+};
+
+export const DATASET_AWSUS_DEM: Dataset = {
```

Comment:
The suggested naming (Mapzen / Copernicus) makes much more sense IMHO, but I'm not sure how much additional work this entails? 

Assigning back the PR in case you decide to change the naming (didn't dive deeply yet), if you decide against it, please assign back.

Reply:
As I said there are 2 reasons I don't like idea of having 4 datasets
-  makeLayers won't work correctly without adding `demInstance` to json capabilites. Based on previous experience it is probably safe to say that this will not happen or at least not any time soon (discussions about adding collectionId, s5 productType, ...)
- I don't see a point of doing it. To me this DEM  (DEM dataset + demInstance combo)  is same use case as S5 datasets with  productType and afaik we have one S5 dataset not 15 of them. Don't get me wrong, it is small change if you ignore issue mentioned in first bullet.

Reply:
I see your point. It seems that the 3 DEM products at services are treated as a single dataset, so let's follow that. Thank you for the explanation, resolving.